lua/telescope/actions/init.lua

```diff
@@ -517,6 +517,28 @@ actions.git_delete_branch = function(prompt_bufnr)
   end
 end
 
+--- Merge the currently selected branch
+---@param prompt_bufnr number: The prompt bufnr
+actions.git_merge_branch = function(prompt_bufnr)
```

Comment:
the branch actions are all very similar. Maybe a local function that takes args_tbl, question, success, failed message. To reduce code duplication

Reply:
Agree, updated.